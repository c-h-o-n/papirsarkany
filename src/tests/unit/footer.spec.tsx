import { render } from "@testing-library/react";
import { expect, test } from "vitest";

import Footer from "~/components/footer";

test("should match snapshot", () => {
  const { container } = render(<Footer />);

  expect(container).toMatchSnapshot();
});
