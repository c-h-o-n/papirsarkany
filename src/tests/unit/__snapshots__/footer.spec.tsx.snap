// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should match snapshot 1`] = `
<div>
  <footer
    class="border-black border-t-4 border-solid bg-sky-50 py-6 text-base-content md:py-10"
  >
    <div
      class="flex flex-col items-center justify-center gap-x-2 gap-y-4 text-center font-medium text-sm md:flex-row"
    >
      <a
        class=""
        href="aszf"
      >
        ÁSZF
      </a>
      <div
        class="d-divider d-divider-neutral d-divider-horizontal hidden md:flex"
      />
      <div
        class="order-last md:order-none"
      >
        © 
        2025
         papirsarkany.hu
      </div>
      <div
        class="d-divider d-divider-neutral d-divider-horizontal hidden md:flex"
      />
      <a
        aria-label="Facebook page"
        href="https://www.facebook.com/profile.php?id=61554743192842"
        rel="noopener noreferrer"
        target="_blank"
      >
        <svg
          class="h-8 w-8"
          fill="none"
          height="800px"
          viewBox="0 0 24 24"
          width="800px"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M2 6C2 3.79086 3.79086 2 6 2H18C20.2091 2 22 3.79086 22 6V18C22 20.2091 20.2091 22 18 22H6C3.79086 22 2 20.2091 2 18V6ZM6 4C4.89543 4 4 4.89543 4 6V18C4 19.1046 4.89543 20 6 20H12V13H11C10.4477 13 10 12.5523 10 12C10 11.4477 10.4477 11 11 11H12V9.5C12 7.567 13.567 6 15.5 6H16.1C16.6523 6 17.1 6.44772 17.1 7C17.1 7.55228 16.6523 8 16.1 8H15.5C14.6716 8 14 8.67157 14 9.5V11H16.1C16.6523 11 17.1 11.4477 17.1 12C17.1 12.5523 16.6523 13 16.1 13H14V20H18C19.1046 20 20 19.1046 20 18V6C20 4.89543 19.1046 4 18 4H6Z"
            fill="#000000"
            fill-rule="evenodd"
          />
        </svg>
      </a>
    </div>
  </footer>
</div>
`;
